presto-cli/src/main/java/io/prestosql/cli/FormatUtils.java

```diff
@@ -165,6 +166,20 @@ public static String formatTime(Duration duration)
         return format("%s:%02d", minutes, seconds);
     }
 
+    public static String formatFinalTime(Duration duration)
+    {
+        long totalMillis = duration.toMillis();
+
+        if (totalMillis >= MINUTES.toMillis(1)) {
+            return formatTime(duration);
+        }
+
+        int seconds = (int) (totalMillis / 1000);
+        int millis = (int) (totalMillis % 1000) / 10;
+
+        return format("%s.%02d", seconds, millis);
```

Comment:
Actually, this could be simplified to:

```suggestion
        return format("%.2f", totalMillis / 1000.0);
```